.github/workflows/verification.yml

```
@@ -0,0 +1,29 @@
+          # verifier than Ubuntu's OpenSSL.
+          - ubuntu-latest
+          - macos-latest
+        script:
+          - install.sh
+          - install.ps1
+      fail-fast: false
+
+    runs-on: ${{ matrix.runs-on }}
+
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)